pkg/metrics/tagkeys.go

```diff
@@ -90,6 +90,10 @@ var (
 	// ResourceKeySyncNamespace groups metrics by the Sync namespace.
 	ResourceKeySyncNamespace, _ = tag.NewKey("configsync_sync_namespace")
 
+	// ResourceKeySyncGeneration groups metrics by the Sync metadata.generation.
+	// This allows matching metrics to a specific Sync configuration.
```

Comment:
Add a brief description of the possible values of this label, so we can have the estimation of the cardinality.

Reply:
It's an int32 that increases any time the sync spec is updated. But since it's included in the reconciler pod spec, it doesn't increase cardinality any more than pod name already does.

Reply:
Ack, thanks for explanation.